ospd/misc.py

```diff
@@ -868,6 +869,9 @@ def filename(string):
                         help='Path to the logging file.')
     parser.add_argument('--version', action='store_true',
                         help='Print version then exit.')
+    parser.add_argument('--niceness', default=NICENESS, type=int,
+                        help='Start the scan with the given niceness. Default 10')
```

Comment:
One additional improvement is possible here (see https://docs.python.org/3/library/argparse.html#help for details)
```suggestion
                        help='Start the scan with the given niceness. Default %(default)s')
```